test/test_test_trixi_include.jl

```diff
@@ -0,0 +1,84 @@
+macro test_trixi_include(expr, args...)
+    local add_to_additional_ignore_content = [r"┌ Warning: Test warning\n└ @ .+\n"]
+    args = append_to_kwargs(args, :additional_ignore_content,
+                            add_to_additional_ignore_content)
+    quote
+        @test_trixi_include_base($(esc(expr)), $(args...))
+    end
+end
+
+@testset verbose=true "@test_trixi_include_base" begin
+    @trixi_testset "basic" begin
+        example = """
+            x = 4
+            """
+
+        mktemp() do path, io
+            write(io, example)
+            close(io)
+
+            @test_trixi_include_base(path)
```

Comment:
```suggestion
            @test_trixi_include_base(path)
            
            @test @isdefined x
            @test x == 4
```
I think, we can also test that with the pure `@test_trixi_include_base`.